test/server/cards/06-WoE/ConductorJarroya.spec.js

```diff
@@ -0,0 +1,29 @@
+describe('Conductor Jărroyă', function () {
+    describe("Conductor Jărroyă's ability", function () {
+        beforeEach(function () {
+            this.setupTest({
+                player1: {
+                    amber: 4,
+                    house: 'ekwidon',
+                    inPlay: ['conductor-jarroya', 'toad', 'shigziso-buggy', 'gauntlet-of-command']
+                },
+                player2: {
+                    inPlay: ['shisnyasi-buggy']
+                }
+            });
+        });
+
+        it('it should ready friendly buggy artifacts', function () {
+            this.player1.useAction(this.shigzisoBuggy);
```

Comment:
I believe the block from 17 to 19 are not needed as they are asserting Shugziso Buggy which you eventually set exhausted in line 21.

Reply:
17-19 deal with the friend Shigziso Buggy, but line 21 deals with the enemy Shisnyasi Buggy.  I believe 17-19 are still needed.

Man these are going to be hard to play with.

Reply:
Yes, it is like playing Lord of the Rings in Sindarin